cloudformation/template.yml

```diff
@@ -17,7 +17,7 @@ Metadata:
           - KeyName
       -
         Label:
-          default: AMIs and Instance Types (Advanced)
+          default: AMIs and Instance Types (Required)
```

Comment:
Related: did you ever wind up making public the AMIs that are referenced as the default values for `CpuAMI` and `GpuAMI`? If not, we should probably remove those defaults so that users don't assume that they can use them.

Reply:
The instance types aren't required, but the AMIs are. I decided not to make the AMI public because it seems like creating the AMI is no more difficult than any of the other parts, and there's the issue of versioning, etc. I'll fix it the way you suggested.